docs/1.getting-started/3.configuration.md

```diff
@@ -133,3 +133,56 @@ Name                                          | Config File             | How To
 | [Stylelint](https://stylelint.io)           | `.stylelintrc.json`     | [More Info](https://stylelint.io/user-guide/configure)
 | [TailwindCSS](https://tailwindcss.com)      |  `tailwind.config.js`   | [More Info](https://tailwindcss.nuxtjs.org/tailwind/config/)
 | [Vitest](https://vitest.dev)                | `vitest.config.ts`      | [More Info](https://vitest.dev/config/)
+
+## Vue Plugin Configuration
+
+If you need to pass some options to `@vitejs/plugin-vue` or `@vitejs/plugin-vue-jsx`, you should also do it using `nuxt.config.ts`. For this purpose, these parameters are present in the Vite configuration:
+- `vite.vue` for `@vitejs/plugin-vue`. Check available options [here](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue).
+- `vite.vueJsx` for `@vitejs/plugin-vue-jsx`. Check available options [here](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue-jsx).
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  vite: {
+    vue: {
+      customElement: true
+    },
+    vueJsx: {
+      mergeProps: true
+    }
+  }
+})
+```
+
```

Comment:
We could also add a description for how to configure `vue-loader`: see https://github.com/nuxt/nuxt/pull/21363.

Reply:
Added it in https://github.com/nuxt/nuxt/pull/21303/commits/b8239651857645eeaf2cbe6d9f37b3e3783b88b1.